executor/extension/logger/progress_logger.go

```diff
@@ -102,8 +104,15 @@ func (l *progressLogger[T]) startReport(reportFrequency time.Duration) {
 		elapsed := time.Since(start)
 		txRate := float64(totalTx) / elapsed.Seconds()
 		gasRate := float64(totalGas) / elapsed.Seconds()
+		days := 0
+		if elapsed.Hours() >= 24 {
+			days = int(elapsed.Hours() / 24)
+			elapsed.Truncate(time.Duration(days * 24))
+			l.log.Noticef(finalSummaryProgressReportFormatWithDays, days, elapsed.Round(time.Second), currentBlock, txRate, gasRate/1e6)
+		} else {
+			l.log.Noticef(finalSummaryProgressReportFormat, elapsed.Round(time.Second), currentBlock, txRate, gasRate/1e6)
```

Comment:
I would only keep one format and remove `else`. Having 0 day is fine imo.

Reply:
The thing is that without this it always logs days. 
`"Elapsed time: 0d15s; current block 1; last interval rate ~10f Tx/s, ~%20 MGas/s"`